### PR TITLE
Prepare 0.4.12

### PR DESCRIPTION
**Pull request by nullr0ute**
A few minor fixes in here, update the spec to create all the config directories.

### CODE REVIEW

File: fido-device-onboard.spec [hunk 1]
```diff
@@ -64,7 +64,20 @@ install -D -m 0644 -t %{buildroot}%{_docdir}/fdo examples/config/*
...
+mkdir -p %{buildroot}%{_sysconfdir}/fdo/stores/manufacturer_keys
+mkdir -p %{buildroot}%{_sysconfdir}/fdo/stores/manufacturing_sessions
+mkdir -p %{buildroot}%{_sysconfdir}/fdo/stores/owner_onboarding_sessions
+mkdir -p %{buildroot}%{_sysconfdir}/fdo/stores/owner_vouchers
+mkdir -p %{buildroot}%{_sysconfdir}/fdo/stores/rendezvous_registered
+mkdir -p %{buildroot}%{_sysconfdir}/fdo/stores/rendezvous_sessions
+mkdir -p %{buildroot}%{_sysconfdir}/fdo/manufacturing-server.conf.d
+mkdir -p %{buildroot}%{_sysconfdir}/fdo/owner-onboarding-server.conf.d
+mkdir -p %{buildroot}%{_sysconfdir}/fdo/rendezvous-server.conf.d
+mkdir -p %{buildroot}%{_sysconfdir}/fdo/serviceinfo-api-server.conf.d
```
Comment (paulwhalen) [inline fido-device-onboard.spec:80]: We should also include `mkdir -p %{buildroot}%{_sysconfdir}/fdo/stores/serviceinfo_api_devices` for the serviceinfo-api-server `device_specific_store_driver` directory

File: fido-device-onboard.spec [hunk 2]
```diff
@@ -93,6 +106,13 @@ Requires: openssl-libs >= 3.0.1-12
 
 %files -n fdo-owner-onboarding-server
 %license LICENSE LICENSE.dependencies
+%dir %{_sysconfdir}/fdo
+%dir %{_sysconfdir}/fdo/keys
+%dir %{_sysconfdir}/fdo/owner-onboarding-server.conf.d
+%dir %{_sysconfdir}/fdo/serviceinfo-api-server.conf.d
+%dir %{_sysconfdir}/fdo/stores
+%dir %{_sysconfdir}/fdo/stores/owner_onboarding_sessions
+%dir %{_sysconfdir}/fdo/stores/owner_vouchers
```
Comment (paulwhalen) [inline fido-device-onboard.spec:115]: Add `%dir %{_sysconfdir}/fdo/stores/serviceinfo_api_devices`

File: fido-device-onboard.spec [hunk 3]
```diff
@@ -145,6 +171,13 @@ Requires: openssl-libs >= 3.0.1-12
 
 %files -n fdo-manufacturing-server
 %license LICENSE LICENSE.dependencies
+%dir %{_sysconfdir}/fdo
+%dir %{_sysconfdir}/fdo/keys
+%dir %{_sysconfdir}/fdo/manufacturing-server.conf.d
+%dir %{_sysconfdir}/fdo/keys
+%dir %{_sysconfdir}/fdo/stores
+%dir %{_sysconfdir}/fdo/stores/manufacturer_keys
+%dir %{_sysconfdir}/fdo/stores/manufacturing_sessions
```
Comment (paulwhalen) [inline fido-device-onboard.spec:180]: `%dir %{_sysconfdir}/fdo/stores/owner_vouchers`

**Review by paulwhalen — CHANGES_REQUESTED** [review]
A couple of changes, otherwise lgtm

File: fido-device-onboard.spec (hunk 2 above)
Comment (nullr0ute) [reply fido-device-onboard.spec:115]: Knew there would be one I missed!

File: fido-device-onboard.spec (hunk 3 above)
Comment (nullr0ute) [reply fido-device-onboard.spec:180]: We need that for both?

Comment (nullr0ute) [reply fido-device-onboard.spec:182]: Updated.

**Review by 7flying — APPROVED** [review]
LGTM

**Pull request merged**